ert_shared/ensemble_evaluator/entity/pydantic/_ensemble_builder.py

```diff
@@ -0,0 +1,186 @@
+import asyncio
+from typing import List, Dict, Coroutine, Any, Iterable
+from pathlib import Path
+import uuid
+import cloudpickle
+
+from ert3.config import (
+    Step as ConfigStep,
+    TransportableCommand,
+    Unix,
+    Function,
+    StagesConfig,
+    EnsembleConfig,
+)
+from ert3.data import (
+    SharedDiskRecordTransmitter,
+    RecordTransmitter,
+    MultiEnsembleRecord,
+    RealisationsToRecordToTransmitter,
+    record_data,
+)
+from ert_shared.ensemble_evaluator.config import EvaluatorServerConfig
+from ._prefect_ensemble import PrefectEnsemble
```

Comment:
One major point about the existing ensemble API (which hasn't been fully realized for ert3) is that it shouldn't have to import this directly.

Reply:
will be resolved in #1694